### DISCUSSION TITLE
Table View - Manually rearranging fields ? or  Fields in same order as in project template

### DISCUSSION THREAD

**Discussion by Flitterby** [marcusolsson/obsidian-projects#369] [Help] [answered]
I'm working on a Gym Tracking project.  In my template, I have the field order as:

Date:
Exercise:
Muscle Group:
Sets:
Reps:
Rest Time:
Weight:
Weight Type:
Time:
Distance:
Intensity:
Notes:

This makes sense for me because then I can include both Cardio and Weight Training items in the same project and make use of the Time, Distance and Intensity fields for Cardio and the other fields for Strength Training.  

Can I manually re-arrange the fields so it's logical for my project or is this something down the track?  Having them in the order of the template would be better as that way it'd be already done from the start.  

I tried drag and drop, but that didn't seem to do anything.  Am I missing something?

**Comment by marcusolsson** [comment, marked as answer]
It's the second most requested feature 😄 

https://github.com/marcusolsson/obsidian-projects/issues/105

I have a few bugs I need to sort out before I'll be able to get to it unfortunately.

**Reply by Flitterby** [reply]
Great.  Knowing it's planned is good.  Sorry I didn't see it in issues.
